scheduler/src/cook/tools.clj

```diff
@@ -963,8 +963,8 @@
   ; Use the already precomputed user->usage map and just aggregate by users to get pool usage.
   (let [pool-usage (reduce (partial merge-with +) (vals user->usage))]
     (->> queue
-         (filter-based-on-pool-quota pool-quota pool-usage)
-         (filter-based-on-user-quota user->quota user->usage))))
+         (filter-based-on-user-quota user->quota user->usage)
+         (filter-based-on-pool-quota pool-quota pool-usage))))
```

Comment:
Could we add a unit or integration test?

Reply:
Done. Currently running through a just-in-case jenkins run.